### DISCUSSION TITLE
Error: nuxt3 with tailwind `resolveAlias`

### DISCUSSION THREAD

**Discussion by dineshbajgain** [nuxt/nuxt#18324] [General]
on `yarn dev`
 `Cannot start nuxt:  Cannot read property 'resolveAlias' of undefined`

tailwind.config.js
`
module.exports = {
  purge: [],
  darkMode: false, // or 'media' or 'class'
  theme: {
    extend: {},
  },
  variants: {
    extend: {},
  },
  plugins: [],
}
`
 nuxt.config.js
`export default {
    buildModules: ['@nuxtjs/tailwindcss']
  }`

**Comment by RobertWade** [comment]
I'd suggest you to jus use [windicss](https://windicss.org/integrations/nuxt.html). It's super easy to set up.

Just pay attention that auto css loading is not supported by nuxt3 yet so your nuxt.config.ts will look like this.

import { defineNuxtConfig } from 'nuxt3';

export default defineNuxtConfig({
  buildModules: ['nuxt-windicss'],
  css: ['virtual:windi.css']
});

**Reply by dineshbajgain** [reply]
why tailwind not worked ?

**Reply by HenrijsS** [reply]
WindiCSS is a good alternative, but that doesn't mean Tailwind should be thrown out the window.

Nonetheless, @dineshbajgain, WindiCSS is now the official recommendation with Nuxt, as you can see on their website:
<img width="428" alt="image" src="https://user-images.githubusercontent.com/49265222/137561853-979b1d00-b2d3-4e4a-803f-1d65c4f0645b.png">

**Comment by Zielgestalt** [comment]
For tailwind status in nuxt3 just follow here: https://github.com/nuxt/framework/discussions/751
